pkg/azdiskschedulerextender/valid-Kube-Config-Path

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
What is this file for?

Reply:
This file gets created when running the scheduler extender unittests with fake kubeclient :D